analytics-integrations/kahuna/src/main/java/com/segment/analytics/internal/integrations/KahunaIntegration.java

```diff
@@ -36,6 +64,7 @@
       throw new IllegalStateException("Kahuna requires the support library to be bundled.");
     }
 
+    trackAllPages = settings.getBoolean("trackAllPages", true);
```

Comment:
Why trackAllPages default to true if not set by user?  This may cause a lot of unintended screen events.